packages/matter.js/src/cluster/server/EventServer.ts

```diff
@@ -28,6 +31,16 @@ export class EventServer<T> {
         this.endpoint = endpoint;
     }
 
+    bindToEventHandler(eventHandler: EventHandler) {
```

Comment:
In fact all data structures are added before we have an eventhandler ( at least in current structure) because eventHandler lives in nInteractionServer and this exists when we add the final structure to it (but then maybe a dev could already have triggered events ... I add a todo for later refactoring.